test/Microsoft.AspNet.Mvc.Razor.Test/Precompilation/Files/TypeInGlobalNamespace.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+public class TypeInGlobalNamespace : ITagHelper
```

Comment:
Make one of these a little more difficult for Roslyn and derive from `TagHelper` instead.